Sources/Honeycomb/Honeycomb.swift

```diff
@@ -262,47 +262,52 @@ public class Honeycomb {
         error: NSError,
         attributes: [String: AttributeValue] = [:],
         thread: Thread?,
+        severity: Severity = .error,
         logger: OpenTelemetryApi.Logger = getDefaultErrorLogger()
     ) {
         let timestamp = Date()
         let type = String(describing: Mirror(reflecting: error).subjectType)
         let code = error.code
+        let domain = error.domain
         let message = error.localizedDescription
 
         var errorAttributes = [
-            "exception.type": type.attributeValue(),
-            "exception.message": message.attributeValue(),
-            "exception.code": code.attributeValue(),
+            "error.type": type.attributeValue(),
+            "error.message": message.attributeValue(),
+            "nserror.code": code.attributeValue(),
+            "nserror.domain": domain.attributeValue(),
         ]
         .merging(attributes, uniquingKeysWith: { (_, last) in last })
 
         if let name = thread?.name {
             errorAttributes["thread.name"] = name.attributeValue()
         }
 
-        logError(errorAttributes, logger, timestamp)
+        logError(errorAttributes, severity, logger, timestamp)
     }
 
     /// Logs an `NSException`. This can be used for logging any caught exceptions in your own code that will not be logged by our crash instrumentation.
     /// - Parameters:
     ///   - exception: The `NSException` itself
     ///   - attributes: Additional attributes you would like to log along with the default ones provided.
     ///   - thread: Thread where the exception occurred. Add this to include additional attributes related to the thread
+    ///   - severity: The severity of the exception. Typically .error or .fatal.
     ///   - logger: Defaults to the Honeycomb error `Logger`. Provide if you want to use a different OpenTelemetry `Logger`
     public static func log(
         exception: NSException,
         attributes: [String: AttributeValue] = [:],
         thread: Thread?,
+        severity: Severity = .error,
         logger: OpenTelemetryApi.Logger = getDefaultErrorLogger()
     ) {
         let timestamp = Date()
-        let type = String(describing: Mirror(reflecting: exception).subjectType)
+        let type = exception.name.rawValue
         let message = exception.reason ?? exception.name.rawValue
 
+        // TODO: Type and name seem wrong here. Which is right?
         var errorAttributes = [
             "exception.type": type.attributeValue(),
             "exception.message": message.attributeValue(),
-            "exception.name": exception.name.rawValue.attributeValue(),
             "exception.stacktrace": exception.callStackSymbols.joined(separator: "\n")
```

Comment:
would be good to reference the semantic conventions from the upstream SDK.

https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetryApi/Trace/SemanticAttributes.swift#L283